### ISSUE TITLE
[MHV SM / Accessibility] Recent recipients page: Zoom/Reflow issue: va-radio-option(s) extends past screen edge

### ISSUE THREAD

**Issue by steveg-IAT** [department-of-veterans-affairs/va.gov-team#119851]
### Point of contact

@steveg-IAT

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

[WCAG 1.4.10 Reflow (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/reflow) requires that web content reflows to fit within a viewport width of 320 CSS pixels when zoomed to 400%

For the "Select care team" step/page within the "Start a new message" flow, with the browser viewport set to 1280px wide @ 400% zoom, the elements listed within `va-radio` with `va-radio-option` extend past the screen edge on the right.

The parent/containing `va-radio` element appears to be contained correctly within the viewport.

(This passes at 200% and 300% zoom)

### Reproduction steps

1. [Stage User #7](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/sm/test-accounts.md)
2. Set browser viewport to 1280px wide (zoom to 400% now or after you get to the target page)
3. From MHV-SM Messages Landing page 
  --  Click: 'Start a new message' 
  -- Click thru: 'Continue to start message' on interstitial page
  -- Arrive at 'Recent care teams' page: 
https://staging.va.gov/my-health/secure-messages/new-message/recent
4. Zoom to 400% 

The issue presents itself.
...

### Proposed solution or next steps

...

### References, articles, or WCAG support

1. [Understanding W3C 1.4.10 Reflow
](https://www.w3.org/WAI/WCAG21/Understanding/reflow)
2.
3.

**Screenshot:**
<img width="1247" height="936" alt="Image" src="https://github.com/user-attachments/assets/ac90be80-7c07-4e87-b726-3b0623e380d2" />

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Voice command
- [x] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
